resources/views/components/layout.blade.php

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://cdn.tailwindcss.com?plugins=forms"></script>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>
+
+    <title>Flight Planner</title>
+</head>
+
+<x-navbar />
```

Comment:
any HTML element must be inside `<body>` tag. Currently this layout is considered malformed. ( browsers fix them on the fly, that's why you dont see it)